src/libs/ajax/DataRepo.ts

```diff
@@ -16,6 +16,7 @@ import { authOpts, fetchDataRepo, jsonBody } from 'src/libs/ajax/ajax-common';
 export type SnapshotBuilderConcept = {
   id: number;
   name: string;
+  code: string;
```

Comment:
The UI seems to work fine, so probably not, but does this create any issues for type compatibility anywhere? I know this is used a lot of places

Reply:
I think if all tests are passing this should be ok. I'll do some more testing